backend/src/main/java/com/bakdata/conquery/io/jackson/serializer/CBlockDeserializer.java

```diff
@@ -27,7 +32,7 @@ public CBlock deserialize(JsonParser p, DeserializationContext ctxt) throws IOEx
 
 		TreeConcept concept = block.getConnector().getConcept();
 
-		if(concept != null && block.getMostSpecificChildren() != null) {
+		if(block.getMostSpecificChildren() != null) {
```

Comment:
wenn das Konzept null ist dann tritt ein NullException später in `concept.getElementByLocalId` auf.